Marlin/src/module/tool_change.cpp

```diff
@@ -274,31 +397,16 @@
                 grabxpos = toolheadposx[tmp_extruder];
 
     /**
-     * 1. Raise Z to give enough clearance
-     * 2. Move to switch position of current toolhead
-     * 3. Unlock tool and drop it in the dock
-     * 4. Move to the new toolhead
-     * 5. Grab and lock the new toolhead
-     * 6. Apply the z-offset of the new toolhead
+	 * Z Lift and Nozzle Offset shift ar defined in caller method to work equal with any Multi Hotend realization
+	 *
```

Comment:
Please configure your editor to use 2 spaces. Tabs are verboten!

Reply:
Sorry used VS now back to NPP that is configured in the correct way.